ts/state/ducks/conversations.ts

```diff
@@ -2093,6 +2096,40 @@ function retryMessageSend(
   };
 }
 
+export function copy(
+  messageId: string
+): ThunkAction<void, RootStateType, unknown, NoopActionType> {
+  return async dispatch => {
+    const { getAbsoluteAttachmentPath } = window.Signal.Migrations;
+    const message = await getMessageById(messageId);
+    if (!message) {
+      throw new Error(`copy: Message ${messageId} missing!`);
+    }
+
+    const body = (message.get('body') || '').trim();
```

Comment:
if you try to copy text in a group where there's an at-mention it'll copy "obj" instead of the mention text. I think if we fix that we can merge.

Reply:
@josh-signal That's right, copying a message with a mention in a group resulted unexpected behavior, for example copying this message: "Hello @mention, how are you?" results with "Hello ￼, how are you?". In my opinion, it should copy the message as is.

I wasn't successful to make it behave like the example i gave above, i couldn't get the mentions as a plain text and i may need some help to implement this behavior.

Reply:
No worries I can take over this PR and fix the conflicts as well if you'd like.

Reply:
@josh-signal Yes please, we could be co-author.

Reply:
🤝